pkg/handlers/handler_base.go

```diff
@@ -0,0 +1,171 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/facebookgo/clock"
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+)
+
+// NewHandlerBase is a constructor for HandlerBase
+func NewHandlerBase(logger *zap.Logger) HandlerBase {
+	return HandlerBase{
+		logger: logger,
+		clock:  clock.New(),
+	}
+}
+
+// HandlerBase is for shared handler utilities
+type HandlerBase struct {
+	logger *zap.Logger
+	clock  clock.Clock
+}
+
+type errorItem struct {
+	Field   string `json:"field"`
```

Comment:
I might add some kind of identifier to classify the type of error for the client to be able to open up and utilize.

Reply:
Yeah, I just don't think we have this classification right now. `errorItem` can definitely be changed as we please